build/cqs.tsv

```diff
@@ -76,6 +80,8 @@ io.airlift:aircompressor	0.10	compile
 io.dropwizard.metrics:metrics-core	3.2.6	compile
 io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
 io.dropwizard.metrics:metrics-graphite	3.2.6	compile
+io.dropwizard.metrics:metrics-jvm	3.2.6	compile
```

Comment:
@jnh5y I am going to skip opening a CQ for this, as we're planning to bump metrics to 4.x. If we don't bump it for whatever reason, don't let me forget to CQ this one before release.